beets/util/confit.py

```diff
@@ -932,6 +932,15 @@ def __init__(self, appname, modname=None):
         self._materialized = False  # Have we read the files yet?
         self._lazy_prefix = []  # Pre-materialization calls to set().
         self._lazy_suffix = []  # Calls to add().
+        self._package_path = _package_path(appname)  # Pre-resolve default source location
+
+    def _add_default_source(self):
+        if self.modname:
+            pkg_path = self._package_path
+            if pkg_path:
+                filename = os.path.join(pkg_path, DEFAULT_FILENAME)
+                if os.path.isfile(filename):
+                    self.add(ConfigSource(load_yaml(filename), filename, True))
```

Comment:
Can we avoid copying most of the implementation of this method? One way, for example, would be to have the Configuration class also use a field called `_package_path`.